### DISCUSSION TITLE
Один префильтр по нескольким полям

### DISCUSSION THREAD

**Discussion by Makhmut2445** [totumonline/totum-ru-issues-and-discussions#149] [Как это сделать?] [answered]
![image](https://user-images.githubusercontent.com/103171127/171817272-27ed166f-53bf-4d2a-ba62-393c761ddbc4.png)
Как сделать чтоб 1 префильтр искал выбранное, через селект, значение во всех полях с ответственными?

**Comment by erpic-ru** [comment, marked as answer]
Теоретически возможный вариант - делаете поле данные в конце , собираете в него все  нужные значения через listcreate и делаете префильтр по этому полю

**Reply by Makhmut2445** [reply]
Я сделал так, не сработало.

**Reply by erpic-ru** [reply]
Незнай, у меня все прекрасно работает. Пример можно импортировать через импорт/экспорт:

[sample.zip](https://github.com/totumonline/totum-ru-issues-and-discussions/files/8831613/sample.zip)

PS: только когда скачаете его надо анзипить перед тем как загружать в Тотум

**Reply by erpic-ru** [reply]
+ через форматирование можно подкрашивать совпадения в столбцах с префильтром
f1=:setFormat(condition: $#nfv = #fl_prefiltr; background: "tomato")

**Reply by erpic-ru** [reply]
А зачем исполнители с типом поля строка, а не селект из справочников?

**Reply by ttm-alexeypolunin** [reply]
Угу — Селект то имеет значение в виде числа (id из таблицы источника). А Ответственные в этом случае написаны буквами.

Если так, то у вас Селект в Префильтре должен иметь параметр `bfield` в котором будут определена колонка с этими ответственными буквами.

**Reply by Makhmut2445** [reply]
Всё, получилось. Я в первый раз сделал через selectrowlist и его взял в kistCreate. Спасибо
